docs/qpileup/index.md

```diff
@@ -1,539 +1,169 @@
 # qpileup
 
-`qpileup` gathers base-level metrics across a whole genome from one or
-more BAM files. It is conceptually similar to samtools or Picard pileup
-in that it parses BAM files to create a summary of information for each
-position in a reference sequence(s). It differs from existing pileup
-implementations in that it captures a much larger range of metrics and
-it makes use of the [HDF5](http://www.hdfgroup.org/HDF5/) data storage
-format to store the data, allowing for smaller files sizes (via 
-compression) and much faster access (via indexing).
+`qpileup` gathers base-level metrics across a whole genome from one or more BAM files. It is conceptually similar to samtools or Picard pileup in that it parses BAM files to create a summary of information for each position in a reference sequence(s). It differs from existing pileup implementations in that it captures a much larger range of metrics and it makes use of the [HDF5](http://www.hdfgroup.org/HDF5/) data storage format to store the data, allowing for smaller files sizes (via compression) and much faster access (via indexing).
 
 ## Installation
 
-qpileup requires java 7 and (ideally) a multi-core machine (5 threads
-are run concurrently) with at least 20GB of RAM.  Download the 
-qpileup tar file and untar into a directory of your choice.
-You should see jar files for qpileup and its dependencies:
+qpileup requires java 8 and (ideally) a multi-core machine. eg. 5 threads with at least 20GB of RAM; 12 threads with at least 30GB and 25 threads with 40GB etc.
+
+* To do a build of qpileup, first clone the adamajava repository.
+  ~~~~{.text}
+  git clone https://github.com/AdamaJava/adamajava
+  ~~~~
+
+*  Then move into the adamajava folder:
+  ~~~~{.text}
+  cd adamajava
+  ~~~~
+
+*  Run gradle to build qsv and its dependent jar files:
+  ~~~~{.text}
+  ./gradlew :qpileup:build
+  ~~~~
+  This creates the qpileup jar file along with dependent jars in the `qpileup/build/flat` folder
 
-~~~~{.text}
-$ tar xvf qpileup.tar.bz2
-x antlr-3.2.jar
-x ini4j-0.5.2-SNAPSHOT.jar
-x jopt-simple-3.2.jar
-x picard-1.110.jar
-x qbamfilter-1.0pre.jar
-x qcommon-0.1pre.jar
-x qio-0.1pre.jar
-x qpicard-0.1pre.jar
-x qpileup-0.1pre.jar
-x sam-1.110.jar
-x jhdfobj.jar
-x jhdf5obj.jar
-x jhdf5.jar
-x jhdf.jar
-~~~~
 
 ## Usage
 
+Before run qpileup, the hdf-java has to be installed and exported, eg.
```

Comment:
Before **_you_** run qpileup, the hdf-java **_library_** has to be...

Reply:
fixed, thanks